index.html

```diff
@@ -309,6 +309,76 @@ <h2 class="heading-primary">About Myself</h2>
         </div>
```

Comment:
Please add all unnecessary files and folders like `.vscode` to `.gitignore`.

Reply:
OK noted